packages/core/integration-tests/test/integration/disable-shared-bundles-false-with-min-bundle-size/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@parcel/bundler-default": {
+    "minBundles": 0,
+    "minBundleSize": 2000,
+    "maxParallelRequests": 100,
+    "disableSharedBundles": false
```

Comment:
I don't know if it makes sense to have "disableSharedBundles" as a config option only. I think a flag could be better for the everyday user. 
For example `parcel build myproject.js --no-shared` would be less confusing. I think most people don't add a bundler config at all (since we have defaults) But let me know what y'all think @gorakong @bdo

Reply:
👍

Reply:
agreed! i think we should also be able to set this through a flag

Reply:
note: decided in Parcel core meeting to keep as a config option only